### ISSUE TITLE
Broken sourcemap

### ISSUE THREAD

**Issue by calle2010** [djhouseknecht/rollup-plugin-version-injector#17]
Hello,
I started to use this plugin just now and, with the default configuration, it works as expected.
But it also produces a warning:

```
(!) Broken sourcemap
https://rollupjs.org/guide/en/#warning-sourcemap-is-likely-to-be-incorrect
Plugins that transform code (such as 'version-injector') should generate accompanying sourcemaps
```

Is this a known issue? Am I missing something in the configuration?

**Comment by djhouseknecht** [comment]
Thanks for the report, @calle2010. I haven't seen this before, but I will look into it. My knee jerk reaction is this won't cause any issues and is just a warning. version-injector is technically "transforming" code which is why I assume that rollup wants vi to also generate source-maps. However, vi is only really doing a find and replace so it shouldn't break any existing source-maps. 

I have been pretty busy so I do apologize for the late reply. It may be a little while before I get a chance to look into this warning more. Please let me know if this starts causing you any issues in the mean time. 

Is your project open source to where I could take a look? If not, could you share some good reproduction steps?

**Comment by ladz** [comment]
Hi! 
I am facing the same issue. I created a [repository](https://github.com/ladz/tmp-rollup-pvi-broken-sourcemap) to reproduce it:
Just install the deps and run `yarn run build`.

![image](https://user-images.githubusercontent.com/1828795/106639319-23c37580-6585-11eb-9c89-0caa41846db0.png)

I tried what is suggested [here](https://rollupjs.org/guide/en/#warning-sourcemap-is-likely-to-be-incorrect). Adding `sourcemap: false` in the plugins rollup config  did not help. Also configuring typescript with `sourceMap: false` did not change anything. To be honest I have no idea if this was the right place to make those changes ;), I just gave it a shot.

Oh, and I also tried to let rollup generate sourcemaps and disabled typescripts - that did not help either.

**Comment by djhouseknecht** [comment]
Thanks for the repo, @ladz! I will try to take a look at this issue this week.

**Comment by calle2010** [comment]
Thanks, both. My project is a small side project at work, so I can't share it. No serious issue on my side. If I find the time I will try to contribute. The test case from @ladz looks exactly like what I see.

**Comment by djhouseknecht** [comment]
This fix should go out with v1.3.1. Let me know if you are still seeing it after upgrading.
